src/pybind/mgr/dashboard/frontend/src/app/shared/components/empty-state/empty-state.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+import { Icons } from '~/app/shared/enum/icons.enum';
+
+@Component({
+  selector: 'cd-empty-state',
+  templateUrl: './empty-state.component.html',
+  styleUrls: ['./empty-state.component.scss']
+})
+export class EmptyStateComponent {
+  @Input()
+  msg: string;
+
+  @Input()
+  btnText: string;
```

Comment:
Let's not be tight with names. Both in Python and JS/TS/Angular worlds, variable names are expressive.
```suggestion
  message: string;

  @Input()
  buttonText: string;
```